legend-engine-executionPlan-execution/pom.xml

```diff
@@ -110,6 +110,11 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-executionPlan-dependencies</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
I dont think we should expose HTTP concepts into the execution plan.  Is there way to design this such that you can expose the data you want while keeping this module clean of this depdency?